src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipServiceTest.java

```diff
@@ -105,6 +110,11 @@ private Transaction buildTransaction() {
     private LimitedPartnershipSubmissionDao createDao() {
         LimitedPartnershipSubmissionDao dao = new LimitedPartnershipSubmissionDao();
         dao.setId(SUBMISSION_ID);
+        dao.setCreatedAt(LocalDateTime.now());
```

Comment:
I don't think you're doing anything with created date and user id so I wouldn't change your test with this data.